add cover alias

alanthecryptodev opened the pull request:
- [x] Add alias for migrate your space

gokaiorg reviewed (commented):
I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/cover-protocol.eth

alanthecryptodev commented:
> I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/cover-protocol.eth

Added the page and strategy
